src/DefinitionResolver.php

```diff
@@ -726,6 +614,7 @@ public function resolveExpressionNodeToType($expr)
             if ($def !== null) {
                 return $def->type;
             }
+            return new Types\Mixed_;
```

Comment:
@felixfbecker This is not related to this PR, I left it here by accident. Should I revert it or let it stay?